ups/meas_extensions_gaap.table

```diff
@@ -0,0 +1,9 @@
+setupRequired(meas_algorithms)
+setupRequired(meas_extensions_gaap)
+setupRequired(afw)
+setupRequired(geom)
+setupRequired(meas_base)
+setupRequired(utils)
+setupRequired(daf_base)
+
+envAppend(PYTHONPATH, ${PRODUCT_DIR}/python)
```

Comment:
Should be `envPrepend` instead of `envAppend` (and if you can remember where you copied this from, we should probably change it, too).

Reply:
I took this from [meas_extensions_convolved](https://github.com/lsst/meas_extensions_convolved/blob/master/ups/meas_extensions_convolved.table) but a Github search shows that this exact line and equivalent statements are in quite a few places, sometimes with `bin` and `lib` folders. Should I open a ticket to change them all?

Reply:
https://jira.lsstcorp.org/browse/DM-29431